client/fingerprint/memory_test.go

```diff
@@ -52,4 +58,5 @@ func TestMemoryFingerprint_Override(t *testing.T) {
 	require := require.New(t)
 	require.NotNil(response.Resources)
 	require.Equal(response.Resources.MemoryMB, memoryMB)
+	require.EqualValues(response.NodeResources.Memory.MemoryMB, memoryMB)
```

Comment:
whats up with the stagger in `Memory.MemoryMB` why 2 x MemoryMB ?

Reply:
Hmm worth considering. It is easy to rename later as these are internal structs, so will consider before 0.9 ships but not in this PR